src/home.js

```diff
@@ -1,21 +1,28 @@
-const homePage = () => {
-    const holder = document.getElementById('holder')
-    holder.setAttribute('id', 'holder')
+
+export default const Home = () => { 
+    holder.setAttribute('id', 'holder')// ?? where is this var coming from?
+    
     const main = document.createElement('div')
-    main.setAttribute('class', 'main')
     const message = document.createElement('p')
-    message.setAttribute('class', 'message')
-    message.innerText = 'Your local fan favorite!'
     const image = document.createElement('img')
-    image.src = 'https://www.idlememe.com/wp-content/uploads/2021/10/salt-bae-meme-idlememe.jpg'
     const message2 = document.createElement('p')
+    
+    //Set attributes
+    main.setAttribute('class', 'main')
+    message.setAttribute('class', 'message')
     message2.setAttribute('class', 'message')
+    
+    //Set contents
+    image.src = 'https://www.idlememe.com/wp-content/uploads/2021/10/salt-bae-meme-idlememe.jpg'
+    message.innerText = 'Your local fan favorite!'
     message2.innerText = 'Serving you since 1890'
 
```

Comment:
Group code by actions rather than objects.
This is simply preference, but it reads easier and makes the code easier to change quickly.